content/rdi/installation/install-k8s.md

```diff
@@ -14,10 +14,36 @@ Throughout the document, the snippets make use of the Kubernetes `kubectl` tool.
 
 ## Prerequisites
 
-- An existing Redis Enterprise cluster version >= 6.2.
-- [RedisGears](https://redis.com/modules/redis-gears/) >= 1.2.5 installed on the cluster. In case it's missing, see [Install RedisGears for Redis Data Integration]({{<relref "/rdi/installation/install-redis-gears.md">}}) to install.
+- An existing Redis Enterprise cluster version >= 6.2.18
```

Comment:
consider having the 6.2.18 as a hugo parameter as it's probably used elsewhere throughout RDI documentation.

Reply:
it's the minimal version of this API so won't change?